pkgs/build-support/bintools-wrapper/ld-wrapper.sh

```diff
@@ -96,6 +96,7 @@ fi
 # Specify the target emulation if nothing is passed in ("-m" overrides this
 # environment variable). Ensures we never blindly fallback on targeting the host
 # platform.
+export LDEMULATION
```

Comment:
This appears to break things. I'm building staging locally and will confirm and find an alternate approach if so.